docs/performance/database-indexes.rst

```diff
@@ -0,0 +1,35 @@
+Database indexes
+================
+
+Indexing tables allow you to decrease fetching time from the database.
+
+As an example, let's take a look at the customers' list.
+
+The default index page is sorted by registration date, to create a table index all you need to do is modify `\App\Entity\Customer\Customer` entity and add the index using annotations. This class can be found at `src/Entity/Customer/Customer.php`. Once open, paste the following code:
```

Comment:
"This class can be found at `src/Entity/Customer/Customer.php`" is it needed? I think we can assume developers would know how to find a class by its FQCN :)

Reply:
`Once open, paste the following code` - it is not exactly true. One can already have some custom code in the class, so we would not like to override it. I would write sth like

```rst
In this file, add indexes attribute into the table configuration:
`@ORM\Table(name="customer", indexes={@Index(name="created_at_index", fields={"created_at"})})`.
Your class should now look like this:
```